docs/debugging-s2i.md

```diff
@@ -0,0 +1,49 @@
+Troubleshooting
+=================
+
+This document contains some tips and suggestions for troubleshooting using the source-to-image command ("sti") to inject source into Docker images.
+
+OpenShift Server
+------------------
+
+The "sti" command of course interfaces with an OpenShift v3 server.  The outputted image is uploaded to the server's internal Docker registry.  If your efforts bring into question how the OpenShift server is operating, check out
```

Comment:
not sure we should be talking about openshift at all here.  openshift pre-reqs s2i, but not the other way around.  s2i itself doesn't upload the image anywhere, it just commits+tags it locally.